### DISCUSSION TITLE
Highlighted yank?

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#5032] [Q&A]
I'd really like to briefly highlight text which I've just yanked, in the vein of machakann/vim-highlightedyank or Neovim's `vim.highlight.on_yank()`. Is this possible?

**Comment by dead10ck** [comment]
No, not currently

**Comment by sudormrfbin** [comment]
In helix you make a selection first and then copy the selection, so whatever is selected currently is what you copied (you could say it's highlighted till you collapse the selection :P).

**Comment by ghost** [comment]
Well that's spam. Anything we can do about that?

**Reply by pascalkuthe** [reply]
It seems targeted four spam posts like that in a short timespan. But these seem to be singleuse accounts so banning won't do much good. I reported them all to github and his them as spam

**Reply by ghost** [reply]
Cheers.
